universal-login-web3/src/services/UIController.ts

```diff
@@ -84,10 +83,10 @@ export class UIController {
   }
 
   appInitialization() {
```

Comment:
Is this needed considering `isLoading === true` by default?

Reply:
it is not neccessary